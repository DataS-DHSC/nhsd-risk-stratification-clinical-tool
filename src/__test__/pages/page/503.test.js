import React from 'react';
import { render } from '@testing-library/react';
import Error503 from '../../../pages/page/503';

const setup = () => render(<Error503 />);

describe('503 page', () => {
  it('should match snapshot', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
