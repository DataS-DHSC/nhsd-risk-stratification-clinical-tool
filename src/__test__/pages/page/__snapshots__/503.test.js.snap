// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`503 page should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-full"
    >
      <h1
        id="error-heading"
      >
        Service unavailable
      </h1>
    </div>
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <p
        style="white-space: pre-wrap;"
      >
        Sorry, but this service is currently unavailable whilst a new version is being released. It will be available again soon. Please try again later.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
